### ISSUE TITLE
Property 'authTokens' does not exist on type 'AuthContextProps'.

### ISSUE THREAD

**Issue by tyteen4a03** [gardner/react-oauth2-pkce#17]
The example references this:

```
  const { authService, authTokens } = useAuth();
```

but `authTokens` doesn't exist on the object returned by `useAuth`.

**Comment by Goethe0** [comment]
yeah, I changed the code to `const authTokens = authService.getAuthTokens()` and `authTokens.idTokens` to `authTokens.id_token`, but still didn't work, the example didn't even load.

**Comment by altenfreelance** [comment]
The example is out of date it seems. @Goethe0 comment above resolved the issue for me. 

`
  const { authService } = useAuth()
  const [authTokens, setAuthTokens] = useState(null)

  useEffect(() => {
    console.log(authTokens)
    if (authService.isAuthenticated()) {
      setAuthTokens(authService.getAuthTokens())
    }
  }, [authService.isAuthenticated()])
`

**Comment by gardner** [comment]
I'll have a look at the example. It hasn't been been updated in a while.
